composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
+        "satooshi/php-coveralls": "^0.7.1"
```

Comment:
because the dependencies (and conflicts) are not managed using `phar`'s as these are using composer (for ref.: https://ci.appveyor.com/project/keradus/php-cs-fixer/build/dev-2529)